src/main/java/spoon/reflect/declaration/CtTypeInformation.java

```diff
@@ -122,7 +122,12 @@
 
 	/**
 	 * Gets the executables declared by this type and by all its supertypes if
-	 * applicable.
+	 * applicable. This method returns:
+	 * <ul>
+	 *     <li>static, instance and default executables</li>
+	 *     <li>Overridden methods</li>
```

Comment:
if it's overridden twice in the hierarchy, does it count for two different elements?